Makefile

```diff
@@ -53,9 +53,12 @@ build.examples.as:
 
 # Use -fstage1 to avoid bugs in the new compiler
 # https://github.com/ziglang/zig/wiki/Self-Hosted-Compiler-Upgrade-Guide#is-it-time-to-upgrade
+%.wasm: %.zig
+	@(cd $(@D); zig build -fstage1)
```

Comment:
off topic:now that 0.10.0 becomes stable, maybe worth trying to  delete`-fstage1` in another PR

Reply:
good idea. there are some unrelated things that break here so we can try it in another PR